sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeServiceAsyncClient.java

```diff
@@ -256,6 +257,7 @@ public String getAccountUrl() {
     public PagedFlux<FileSystemItem> listFileSystems(ListFileSystemsOptions options) {
         try {
             return blobServiceAsyncClient.listBlobContainers(Transforms.toListBlobContainersOptions(options))
+//                .onErrorMap(ex -> DataLakeImplUtils.transformBlobStorageException((BlobStorageException) ex))
```

Comment:
Can you open an issue for this to track it and reference the issue open against PagedIterable and PagedFlux?

Reply:
Created here https://github.com/Azure/azure-sdk-for-java/issues/6808